### DISCUSSION TITLE
avatar图片放哪里?

### DISCUSSION THREAD

**Discussion by duchenpaul** [theme-shoka-x/hexo-theme-shokaX#24] [Q&A] [answered]
在文件系统里面, avatar图片放在哪里? 在shoka里面, 是放在`./source/_data/images/`, shokaX似乎改变了

**Comment by zkz098** [comment, marked as answer]
和shoka差不多，把images改成assets就行

**Reply by duchenpaul** [reply]
Thanks, it is good to mention it in the _config.yml
